### DISCUSSION TITLE
Meaning of Cookie Affected column in the CSV reports generate by CLI and the report download from the plugin

### DISCUSSION THREAD

**Discussion by dhineshbabu95** [GoogleChromeLabs/ps-analysis-tool#592] [Support Forum] [answered]
The column only consists of 2 values "yes" or "no". 

We know that all Third Party cookies will be affected so, why does the column show "no" for some of the Third party cookies in the extracted CSV. If that was not the purpose of the column what does it define?

**Comment by milindmore22** [comment, marked as answer]
Hello @dhineshbabu95 

A similar question has been asked previously, please check the answer for the discussion [#569 ](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/569#discussioncomment-8858541)

Additionally, please note that the cookie Affected column will be renamed to "issues" in an upcoming release, please check https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/580
